examples/block_author.ts

```diff
@@ -0,0 +1,42 @@
+import * as C from "http://localhost:5646/@local/mod.ts"
+import {
+  Session,
+  System,
+} from "http://localhost:5646/@local/proxy/wss:rpc.polkadot.io/pallets/mod.ts"
+import { $preDigest } from "http://localhost:5646/@local/proxy/wss:rpc.polkadot.io/types/sp_consensus_babe/digests.ts"
+import {
+  $digestItem,
+  DigestItem,
+} from "http://localhost:5646/@local/proxy/wss:rpc.polkadot.io/types/sp_runtime/generic/digest.ts"
+import * as U from "http://localhost:5646/@local/util/mod.ts"
+
+const blockHeader = C.chain.getHeader(C.polkadot)()
+const header = U.throwIfError(await blockHeader.run())
+const digestedLogs = header.digest.logs.map(C.hex.decode).map($digestItem.decode.bind($digestItem))
```

Comment:
```suggestion
const digestLogs = header.digest.logs.map(x => $digestItem.decode(C.hex.decode(x)))
```

Reply:
easier to read than map chaining eh? its what i had originally

Reply:
Easier to read and more efficient ;)